docs/download.md

```diff
@@ -199,8 +218,10 @@ Four binaries are currently available:
 
 @tab macOS
 
-Two binaries will be listed but the only one you will likely want is the
-`dmg` release.
+Two binaries are provided:
+
+- `dmg` standard installer image
```

Comment:
```suggestion
- `dmg` standard installer image (You likely want this one)
```

Reply:
So I removed this wording because I think this just needs to be factual, it was important when it was the only option but I think anyone looking down here knows and understands what they are doing rather than doing the "proper" links.

Reply:
Yeah, the .zip is really just a different compression method/archive format around the `.app` bundle if I recall correctly.

The `.dmg` opens to one of those fancy styled folders that gives you the nice "drag and drop onto the Applications folder" flow in Finder. But the .zip can be extracted and dragged/dropped onto `/Applications` or `cp`ed there from the CLI, whatever. Mmhmm.

I agree with removing the "you probably want the dmg". text.